environments/se/core/src/main/java/org/jboss/weld/environment/se/WeldContainer.java

```diff
@@ -209,6 +210,9 @@ private void complete() {
         this.creationalContext = beanManager().createCreationalContext(null);
         this.instance = beanManager().getInstance(creationalContext);
         this.event = beanManager().event();
+    }
+
+    private void fireContainerInitializedEvent() {
         beanManager().fireEvent(new ContainerInitialized(id), Initialized.Literal.APPLICATION);
         WeldSELogger.LOG.weldContainerInitialized(id);
```

Comment:
Hm, don't move the message - it should be logged before the event, otherwise it will never appear if a `ContainerInitialized` observer blocks...